console/src/view/mod.rs

```diff
@@ -93,12 +104,28 @@ impl View {
                             ));
                         }
                     }
+                    key!(F(2)) => {
```

Comment:
somewhat on the fence about using function keys here; on some keyboard layouts they might require a second keypress, be mapped to other commands by the user, or be on the touch bar on some macbooks...part of me wants to suggest `r` for resources and `t` for tasks, instead. but, i could be convinced otherwise...

we could always add support for custom keymaps later!